internal/app/dogma/go.mod

```diff
@@ -4,7 +4,6 @@ require (
 	github.com/fhs/go-netrc v1.0.0
 	github.com/urfave/cli v1.20.0
 	go.linecorp.com/centraldogma v0.0.0-20181220074123-455505acb33f
```

Comment:
:+1: Please update with latest version of central dogma with http/2. I think it's: `a046020` 😄

Reply:
Oops, fixed. Thank you~!